storage/cloud.py

```diff
@@ -696,6 +696,28 @@ def cancel_chunked_upload(self, uuid, storage_metadata):
         for chunk in self._chunk_list_from_metadata(storage_metadata):
             self.remove(chunk.path)
 
+    def clean_partial_uploads(self, deletion_date_threshold):
+        self._initialize_cloud_conn()
+        path = self._init_path("uploads")
```

Comment:
Does this need a trailing `/`?

Reply:
I'm not actually seeing a trailing `/`, i believe i did have it in a previous commit but removed it afterword.